css-gaps-1/Overview.bs

```diff
@@ -0,0 +1,601 @@
+<pre class='metadata'>
+Title: CSS Gap Decorations Module Level 1
+Shortname: css-gaps
+Level: 1
+Status: ED
+Group: csswg
+Work Status: exploring
+Repository: w3c/csswg-drafts
+URL: https://drafts.csswg.org/css-gaps-1
+Editor: Kevin Babbitt, Microsoft, https://github.com/kbabbitt, w3cid 124689
+Abstract: This module introduces several properties to add row and column gap decorations to container layout types such as grid and flex.
+</pre>
+<pre class='anchors'>
+urlPrefix: https://drafts.csswg.org/css-grid-3/; type: dfn; spec: CSS-GRID-3; text: masonry container
+</pre>
+
+Introduction {#intro}
+=====================
+
+	<em>This section is not normative.</em>
+
+	[[CSS-MULTICOL-1#column-gaps-and-rules]] allows for rules to be drawn
```

Comment:
Should we note that this will eventually be deprecated in favor of this specification?

Reply:
Yes, but probably not here since this section is not normative. Added a sentence to the section with the normative property definitions.